tests/resources/mysql_test_tables.sql

```diff
@@ -134,6 +134,30 @@ INSERT INTO `pso_data_validator`.`dvt_datetime_id` VALUES
 ('2020-01-01 12:00:00', 'Row 1'), ('2020-02-01 12:00:00', 'Row 2'), ('2020-03-01 12:00:00', 'Row 3'),
 ('2020-04-01 12:00:00', 'Row 4'), ('2020-05-01 12:00:00', 'Row 5');
 
+DROP TABLE IF EXISTS `pso_data_validator`.`dvt_varchar_id`
+CREATE TABLE `pso_data_validator`.`dvt_varchar_id`
+(   id          varchar(15) NOT NULL
+,   other_data  varchar(100)
+) OPTIONS (description='Integration test table used to test variable length string, trailing blanks are significant');
+INSERT INTO `pso_data_validator`.`dvt_varchar_id` VALUES
+('DVT-key-1', 'Row 1'),
+('DVT-key-2', 'Row 2'),
+('DVT-key-3', 'Row 3'),
+('DVT-key-4 ', 'Row 4'),
+('DVT-key-5', 'Row 5');
+
+DROP TABLE IF EXISTS `pso_data_validator`.`dvt_fixed_char_id`
```

Comment:
We already have `dvt_char_id` for several systems, this new table is a duplicate.